### PR TITLE
Updated first steps in portuguese language (pt-br)

### PR DESCRIPTION
**Pull request by nbfontana**
I have updated the first steps index.html file with translated content from more recent samples.
The content of the file was too outdated.

> No issue was closed.

### CODE REVIEW

**Comment by abose** [comment]
Thanks @nbfontana .
I have targetted this for 1.7. 
Tagging for review  @Rynaro @eliezerb @brunnopleffken @rodrigost23

**Comment by brunnopleffken** [comment]
Nice @nbfontana!! And it's really well translated, including expressions, just a few minor observations:

- I believe _"um moderno editor de código open-source que entende **de** web design"_ sounds better, what do you think?
- In _What is Brackets?_ the `<em>` tag is missing.
- In line 68, the word _JavaScript_ is all lower case (BTW, JavaScript is an UpperCamelCased word).

And that's it! Ready to go! :D

File: samples/pt-br/Primeiros Passos/index.html [hunk 1]
```diff
@@ -14,46 +14,68 @@
...
     
     <!--
-    O QUE É BRACKETS?
+        O QUE É O BRACKETS?
     -->
+        
     <p>
-        <em>Você está olhando para uma versão precoce de Brackets.</em>
-        De muitas maneiras, Brackets é um tipo diferente de editor. Uma diferença notável é que este editor é escrito em JavaScript. Assim, enquanto Brackets poderia não estar pronto para seu uso no dia-a-dia ainda, estamos usando-o todos os dias para criar Brackets.
+        Brackets é um editor diferente e tem algumas caracteristicas únicas, como a Edição Rápida (Quick Edit), Visualização ao vivo (Live Preview) e muitas outras que você não encontrará em outros editores. E Brackets é escrito em Javascript, HTML e CSS. Isso significa que a maioria de vocês que utilizam o Brackets possuem as habilidades necessárias para modificar e extender o editor. Na verdade, usamos o Brackets todos os dias para desenvolver o Brackets. Para saber mais sobre como utilizar os principais recursos, continue lendo.
```
Comment (eliezerb) [inline samples/pt-br/Primeiros Passos/index.html]: Some minor changes:
* caracteristicas -> características
* extender -> estender
* "E Brackets" -> "E o Brackets"

File: samples/pt-br/Primeiros Passos/index.html [hunk 2]
```diff
@@ -14,46 +14,68 @@
...
+        COMECE COM SEUS PRÓPRIOS ARQUIVOS
+    -->
+    
+    <h3>Projetos no Brackets</h3>
+    <p>
+        Para editar seu próprio código utilizando o Brackets, basta abrir a pasta contendo seus arquivos. Brackets trata a pasta atualmente aberta como um "Projeto", recursos como Code Hints, Live Preview e Quick Edit usam apenas os arquivos da pasta atualmente aberta.
+    </p>
+        
+    <samp>
+        Assim que você estiver pronto para sair desse projeto exemplo e editar seu próprio código, você pode usar o menu na barra lateral para trocar de pasta. Nesse momento, o menu diz "Primeiros Passos" - que é pasta que possui o arquivo que vocês está olhando agora mesmo. Clique no menu e escolhar "Abrir pasta..." para abrir sua própria pasta.
```
Comment (eliezerb) [inline samples/pt-br/Primeiros Passos/index.html]: Some minor changes:

* "Clique no menu e escolhar" -> "Clique no menu e escolha"
* "que é pasta" -> "que é a pasta"
*  "que vocês está" ->  "que você está"

File: samples/pt-br/Primeiros Passos/index.html [hunk 3]
```diff
@@ -14,46 +14,68 @@
...
+        Você também pode usar o menu para voltar a pastas que você abriu anteriormente, incluindo esse projeto de exemplo.
```
Comment (eliezerb) [inline samples/pt-br/Primeiros Passos/index.html]: " Você também pode usar" -> "Também é possível utilizar"

**Comment by eliezerb** [comment]
Very nice @nbfontana! I will let some minor suggestions as notes, you can have a look on those and change what you think it's necessary.

I just would like to have a second thought in some places where you say:

"_Brackets trata a pasta atualmente_", where you could maybe say that "_O Brackets [...]_", at least for me sounds much more natural to read. The same happens in other places. What you guys think?

Once more, congrats for the great job!

File: samples/pt-br/Primeiros Passos/index.html [hunk 4]
```diff
@@ -14,46 +14,68 @@
...
+        Faça uma modificação no seu CSS, aperte <kbd>ESC</kbd> e você está de volta a edição do HTML, ou apenas deixe as regras CSS abertas e elas se tornarão parte do seu editor HTML. Se você apertar <kbd>ESC</kbd> fora do quick inline editor, todos eles irão se recolher. Quick Edit também encontrará regras definidas em arquivos LESS e SCSS, incluindo regras aninhadas.
     </p>
     
     <samp>
-        Quer vê-lo em ação? Coloque o cursor sobre o tag <!-- <samp> --> acima e pressione <kbd>Cmd/Ctrl + E</kbd>. Você deverá ver um editor rápido de CSS aparecer acima. À direita, você verá uma lista de regras CSS que estão relacionadas com esta tag.Simplesmente role as regras com <kbd>Alt + Up/Down</kbd> para encontrar o que deseja editar.
+        Quer vê-lo em ação? Coloque o cursor sobre o tag <!-- <samp> --> acima e pressione <kbd>Cmd/Ctrl + E</kbd>. Você deverá ver um editor rápido de CSS aparecer acima, mostrando as regras CSS que estão aplicadas a ele. Quick Edit funciona em classes e atribudos id também. Você pode usa-lo com seus arquivos LESS e SCSS também.
```
Comment (eliezerb) [inline samples/pt-br/Primeiros Passos/index.html]: * "Você pode usa-lo" -> "Você pode usá-lo "

I think "_a tag_" sounds better than "_o tag_". What you guys think?

File: samples/pt-br/Primeiros Passos/index.html [hunk 5]
```diff
@@ -69,15 +97,31 @@
...
+        Atualmente, Brackets suporta Visualização ao Vivo (Live Preview) apenas para HTML e CSS. Entretanto, na versão atual, mudanças nos arquivos JavaScript são automaticamente carregadas quando você salva. Estamos trabalhando no suporte à Visualização ao Vivo para JavaScript. Visualizações ao vivo atualmente só são possíveis com Google Chrome, mas nós esperamos trazer essa funcionalidade para os principais navegadores no futuro.
     </p>
     
+    <h3>Quick View</h3>
+    <p>
+        Para aqueles de nós que ainda não memorizaram os códigos de cores equivalentes para HEX e RGB, Brackets facilita ver exatamente qual cor está sendo utilizada. Tanto no CSS quanto no HTML, basta passar o mouse por cima de qualquer valor de cor ou gradiente que o Brackets irá mostrar um pré visualização da cor/gradiente. O mesmo vale para imagens: passe o mouse por cima do link da imagem no Brackets  e ele irá mostrar uma pré visualização dessa imagem.
```
Comment (eliezerb) [inline samples/pt-br/Primeiros Passos/index.html]: * "irá mostrar um pré visualização" -> "irá mostrar uma pré-visualização"
* "mostrar uma pré visualização dessa imagem" -> "mostrar uma pré-visualização dessa imagem"

File: samples/pt-br/Primeiros Passos/index.html [hunk 6]
```diff
@@ -69,15 +97,31 @@
...
+    </p>
+        
+    <samp>
+        Tente o Quick View você mesmo, passe o cursos na tag <!-- <body> --> no topo desse documento e aperte <kbd>Cmd/Ctrl + E</kbd> para abrir o editor rápido de CSS. Agora simplesmente passe o mouse por cima de qualquer um dos valores de cor no CSS. Você também pode ver isso em ação em gradientes abrindo um editor rápido de CSS na tag <!-- <html> --> e passando o mouse por qualquer um dos dos valores de background image. Tente também a pré visualização de imagens, coloque o mouse sobre o link de screenshot incluido anteriormente nesse documento.
```
Comment (eliezerb) [inline samples/pt-br/Primeiros Passos/index.html]: * "passe o cursos" -> "passe o cursor"
* "background image" -> "`background-image`"
* " pré visualização" -> "pré-visualização"
* "incluido" -> "incluído"

File: samples/pt-br/Primeiros Passos/index.html [hunk 7]
```diff
@@ -69,15 +97,31 @@
...
+    </samp>
+        
+    <h3>Precisa de algo mais? Tente uma extensão!</h3>
+    <p>
+        Além de todas as coisas boas contruidas no Brackets, nossa grande comunidade de desenvolvedores de extensões tem construido centenas de extensões que adicionam funcionalidades uteis ao editor. Se tem algo que você precisa que o Brackets não oferece, mais do que provavelmente alguem já construiu uma extensão para isso. Para pesquisar na lista de extensões disponíveis, escolha <strong>Arquivo > Gerenciador de Extensões...</strong> e clique na aba "Disponíveis". Quando encontrar uma extensão que você quer, apenas clique no botão "Instalar" ao lado.
```
Comment (eliezerb) [inline samples/pt-br/Primeiros Passos/index.html]: "contruidas" -> "construídas"
"construido" -> "construído"
"alguem" -> "alguém"

File: samples/pt-br/Primeiros Passos/index.html [hunk 8]
```diff
@@ -69,15 +97,31 @@
...
+        
     <!--
-        DEIXE-NOS CONHECER O QUE VOCÊ PENSA
+        DEIXE-NOS SABER O QUE VOCÊ PENSA
     -->
+        
     <h2>Envolva-se</h2>
     <p>
-        Brackets é um projeto open-source. Desenvolvedores web de todo o mundo estão a contribuir para criar um editor de código melhor. Diga-nos o que você pensa, partilhe as suas ideias ou contribua diretamente para o projeto.
+        Brackets é um projeto open-source. Desenvolvedores web de todo o mundo estão a contribuir para criar um editor de código melhor. Muitos outros estão desenvolvendo extensões que expandem as capacidades do Brackets.
```
Comment (eliezerb) [inline samples/pt-br/Primeiros Passos/index.html]: "expandem" -> "espandem"

File: samples/pt-br/Primeiros Passos/index.html (hunk 4 above)
Comment (brunnopleffken) [inline samples/pt-br/Primeiros Passos/index.html]: Indeed, _"o Brackets"_ sounds better than just _"Brackets"_ without an article before it. The same with "a tag", although an English word, it's treated as a feminine word in Portuguese.

File: samples/pt-br/Primeiros Passos/index.html [hunk 9]
```diff
@@ -69,15 +97,31 @@
...
     <!--
-        DEIXE-NOS CONHECER O QUE VOCÊ PENSA
+        DEIXE-NOS SABER O QUE VOCÊ PENSA
     -->
+        
     <h2>Envolva-se</h2>
     <p>
-        Brackets é um projeto open-source. Desenvolvedores web de todo o mundo estão a contribuir para criar um editor de código melhor. Diga-nos o que você pensa, partilhe as suas ideias ou contribua diretamente para o projeto.
+        Brackets é um projeto open-source. Desenvolvedores web de todo o mundo estão a contribuir para criar um editor de código melhor. Muitos outros estão desenvolvendo extensões que expandem as capacidades do Brackets. 
+        Diga-nos o que você pensa, partilhe as suas ideias ou contribua diretamente para o projeto.
```
Comment (eliezerb) [inline samples/pt-br/Primeiros Passos/index.html]: "_Compartilhe_" sounds better than "_partilhe_". What do you think?

File: samples/pt-br/Primeiros Passos/index.html [hunk 10]
```diff
@@ -61,6 +83,12 @@
         Brackets vai abrir uma <em>conexão ao vivo</em> com o seu navegador local e vai empurrar atualizações CSS enquanto você digita! Você já deve estar fazendo alguma coisa como esta hoje com ferramentas baseadas em navegador, mas com Brackets
         não há necessidade de copiar e colar o CSS final de volta para o editor. Seu código é executado no navegador, mas vive em seu editor!
     </p>
+        
+    <h3>Destaque ao vivo de elementos HTML e regras CSS</h3>
+    <p>
+        Brackets facilita ver como as mudanças no seu HTML e CSS irão afetar a página.
+        Quando seu cursor está em uma regra CSS, Brackets irá destacar todos os elementos afetados no navegador. Similarmente, quando estiver editando um arquivo HTML, Brackets irá destacar o elemento HTML correpondente no navegador.
```
Comment (eliezerb) [inline samples/pt-br/Primeiros Passos/index.html]: "correpondente" -> "correspondente"

File: samples/pt-br/Primeiros Passos/index.html [hunk 11]
```diff
@@ -14,46 +14,68 @@
...
     
     <a href="screenshots/quick-edit.png">
         <img alt="Um screenshot mostrando o editor rápido de CSS" src="screenshots/quick-edit.png" />
     </a>
+        
+    <p>
+        Você também pode usar o mesmo atalho para editar outras coisas - como funções em javascript, cores e funções de tempo de animação - e nós estamos adicionando mais e mais o tempo todo.
+    </p>
+    <p>
+        Por enquando inline editors não podem ser aninhados, então você só pode usar Quick Edit enquando o cursor estiver em um editor de tamanho máximo.
```
Comment (eliezerb) [inline samples/pt-br/Primeiros Passos/index.html]: * "Por enquando" -> "Por enquanto"
* "enquando o cursor" -> "enquanto o cursor"

**Comment by nbfontana** [comment]
Thank you for the review and for highlighting my portuguese and typing errors **@eliezerb** and **@brunnopleffken**.

I have made an effort to translate the content without consulting any kind of translation tool, and i was going review the translation before submiting the pull request, but i forgot... I'm glad you guys found all those mistakes. Next time i'll be more carefull!

I will work on the suggested corrections.

**Comment by eliezerb** [comment]
@nbfontana It's ok! Thanks for the efforts! Let us know when we can have another look and complete the review process to make this awesome translation available to everyone :+1:

**Comment by nbfontana** [comment]
You guys are free to review again

File: samples/pt-br/Primeiros Passos/index.html [hunk 12]
```diff
@@ -26,7 +26,7 @@
     -->
         
     <p>
-        Brackets é um editor diferente e tem algumas características únicas, como a Edição Rápida (Quick Edit), Visualização ao vivo (Live Preview) e muitas outras que você não encontrará em outros editores. E o Brackets é escrito em JavaScript, HTML e CSS. Isso significa que a maioria de vocês que utilizam o Brackets possuem as habilidades necessárias para modificar e estender o editor. Na verdade, usamos o Brackets todos os dias para desenvolver o Brackets. Para saber mais sobre como utilizar os principais recursos, continue lendo.
+        O Brackets é um editor diferente e tem algumas características únicas, como a Edição Rápida (Quick Edit), Visualização ao vivo (Live Preview) e muitas outras que você não encontrará em outros editores. E o Brackets é escrito em JavaScript, HTML e CSS. Isso significa que a maioria de vocês que utilizam o Brackets possuem as habilidades necessárias para modificar e estender o editor. Na verdade, usamos o Brackets todos os dias para desenvolver o Brackets. Para saber mais sobre como utilizar os principais recursos, continue lendo.
```
Comment (eliezerb) [inline samples/pt-br/Primeiros Passos/index.html]: Like @brunnopleffken commented earlier, the original page says:

"<em>Brackets is a different type of editor.</em>
 Brackets has some unique features like Quick Edit, [...]"

Maybe you could maintain the same structure with something like:

"<em>O Brackets é um editor diferente.</em>
O Brackets tem algumas características únicas, como a Edição Rápida [...]"

File: samples/pt-br/Primeiros Passos/index.html [hunk 13]
```diff
@@ -77,32 +77,32 @@
         
     <h3>Visualize alterações no HTML e CSS ao vivo no navegador</h3>
     <p>
-        Você sabe aquela "dança salvar/recarregar" que temos feito há anos? Aquela onde você faz mudanças no seu editor, clica em salvar, alterna para o navegador e então recarrega a página para finalmente ver o resultado? Com Brackets, você não precisa fazer essa dança.
+        Você sabe aquela "dança salvar/recarregar" que temos feito há anos? Aquela onde você faz mudanças no seu editor, clica em salvar, alterna para o navegador e então recarrega a página para finalmente ver o resultado? Com o Brackets, você não precisa fazer essa dança.
     </p>
     <p>
-        Brackets vai abrir uma <em>conexão ao vivo</em> com o seu navegador local e vai empurrar atualizações CSS enquanto você digita! Você já deve estar fazendo alguma coisa como esta hoje com ferramentas baseadas em navegador, mas com Brackets
+        O Brackets vai abrir uma <em>conexão ao vivo</em> com o seu navegador local e vai empurrar atualizações CSS enquanto você digita! Você já deve estar fazendo alguma coisa como esta hoje com ferramentas baseadas em navegador, mas com o Brackets
```
Comment (eliezerb) [inline samples/pt-br/Primeiros Passos/index.html]: "O Brackets vai abrir uma <em>conexão ao vivo</em> com o seu navegador local e vai **empurrar** atualizações CSS enquanto você digita!"

I think **empurrar** is a little bit out of context in here, maybe **enviar as atualizações no CSS**, what you guys think?

**Comment by eliezerb** [comment]
@nbfontana If you don't mind, can you please check those last two comments? I promise they are the last ones, then we're good to go!

File: samples/pt-br/Primeiros Passos/index.html (hunk 13 above)
Comment (nbfontana) [inline samples/pt-br/Primeiros Passos/index.html]: Totally agree

**Comment by nbfontana** [comment]
Your turn @eliezerb

**Comment by eliezerb** [comment]
@abose looks good to me now! Congrats for the great work @nbfontana :)

**Comment by abose** [comment]
Thanks @nbfontana for contributing @eliezerb @brunnopleffken For reviewing.
Merging. :fireworks:

**Pull request merged**